simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/FOSS5xPerfMedium.scala

```diff
@@ -0,0 +1,500 @@
+package com.puppetlabs.gatling.node_simulations
+import com.puppetlabs.gatling.runner.SimulationWithScenario
+import org.joda.time.LocalDateTime
+import org.joda.time.format.ISODateTimeFormat
+import java.util.UUID
+
+import scala.concurrent.duration._
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+// import io.gatling.jdbc.Predef._
+
+class FOSS5xPerfMedium extends SimulationWithScenario {
+
+// 	val httpProtocol = http
+// 		.baseURL("https://${node}:8140")
+
+	val reportBody = ELFileBody("FOSS5xPerfMedium_0127_request.txt")
+
+	val headers_0 = Map("X-Puppet-Version" -> "5.0.0")
+	val headers_1 = Map(
+		"X-Puppet-Version" -> "5.0.0",
+		"Accept" -> "application/json",
+		"Content-Type" -> "application/json",
+		"Connection" -> "close")
+// val uri1 = "https://${node}:8140/puppet/v3"
+
+	val chain_0 = exec(http("node")
+			.get("/puppet/v3/node/${node}?environment=production&transaction_uuid=122098b0-4ca4-4b70-ba22-a6fdf331b352&fail_on_404=true")
+			.headers(headers_0))
```

Comment:
Hmm, the `node` and `file_meta` requests should also be including `Accept: application/json` and `Accept-Encoding: gzip` like the `catalog` and `report` ones do.  I think we should add those into `headers_0`, like:

```scala
val headers_0 = Map(
		"X-Puppet-Version" -> "5.0.0",
		"Accept" -> "application/json",
                "Accept-Encoding" -> "gzip;q=1.0,deflate;q=0.6,identity;q=0.3")
```